### PR TITLE
disable default for number insert/update

### PR DESCRIPTION
**Pull request by Idokah**
Change the defaults to be null for all the types but system Fields,
add two tests for insert, update numeric fields.
Fix mysql_utils bug on `patchObjectField`

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -44,7 +44,6 @@ jobs:
                    "mssql", "mssql17",
                    "mongo", "mongo4",
                    "firestore",
-                   "airtable",
```
Comment (MXPOL) [inline .github/workflows/main.yml:47]: Why did you delete this?

**Review by MXPOL — APPROVED** [review]

File: .github/workflows/main.yml (hunk 1 above)
Comment (Idokah) [reply .github/workflows/main.yml:47]: We never really supported this, all the tests are running on some mock implementation that we maintain, so in every new test I need to add more complexity to the mock or not support this test.
Anyway, I talked with @michaelir, and we decided to let it go for now.
